plugins/jira/docs/issue-types/epic.md

```diff
@@ -0,0 +1,498 @@
+<!-- Generated by Claude Sonnet 4.5 (Anthropic) - December 2025 -->
+
+# Epic Guide
+
+Guide on Jira Epics: what they are, how to use them, and best practices.
+
+## Using Templates
+
+See [Template Reference Guide](../../templates/README.md#template-reference-guide) for complete template→guide mapping.
+
+**Create an epic:**
+```bash
+# Auto-selects appropriate template
+/jira:create epic OCPEDGE "Multi-cluster metrics aggregation for ROSA HCP"
```

Comment:
Should this obfuscate the project or update the comment so it knows this is just an example vs all epics should be made under OCPEDGE?

Reply:
Good catch. :) I'll run a scan for team-specific examples.